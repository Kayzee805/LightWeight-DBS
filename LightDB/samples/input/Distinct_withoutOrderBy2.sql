SELECT DISTINCT S2.C, S1.B FROM Sailors S1, Sailors S2 WHERE S1.B <= S2.C;
