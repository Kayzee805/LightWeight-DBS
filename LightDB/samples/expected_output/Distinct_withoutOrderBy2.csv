105,100
200,200
200,100
400,200
400,100
400,300
500,200
500,100
500,300
